### ISSUE TITLE
modify backward loss weight

### ISSUE THREAD

**Issue by westpilgrim** [xingyizhou/CenterNet#97]
how to modify backward loss weight? I only see the forward loss weight, in script CenterNet/src/lib/trains/multi_pose.py:    
loss = opt.hm_weight * hm_loss + opt.wh_weight * wh_loss + \
           opt.off_weight * off_loss + opt.hp_weight * hp_loss + \
           opt.hm_hp_weight * hm_hp_loss + opt.off_weight * hp_offset_loss
but I did not see these weights are applied in backward? where did you apply them?

**Comment by westpilgrim** [comment]
if I cannot find where the loss weights are applied in back warding, I would not be able to add new loss and adjust loss weights

**Comment by westpilgrim** [comment]
solved the problem . auto grad
